app/models/manageiq/providers/automation_manager/credential.rb

```diff
@@ -0,0 +1,2 @@
+class ManageIQ::Providers::AutomationManager::Credential < Authentication
```

Comment:
Since this is still on the ManageIQ side of the fence, wouldn't it make more sense to name this `ManageIQ::Providers::AutomationManager::Authentication`?